dolphin/common/constants.py

```diff
@@ -48,3 +49,21 @@ class StorageType(object):
     FILE = 'file'
 
     ALL = (BLOCK, FILE)
+
+
+class ResourceType(IntEnum):
+    STORAGE_DEVICE = 0
+    POOL = 1
+    VOLUME = 2
+    ACCESS_INFO = 3
+    DISK = 4
+    ALERT_SOURCE = 5
```

Comment:
ACCESS_INFO and ALERT_SOURCE are not part of task manager .How do we check if there is some activty happening for these resources like ( update access-info , or remove alert source ) ??

Reply:
Removed all of the rest except STORAGE_DEVICE, POOL and VOLUME.